src/Microsoft.ComponentDetection.Detectors/maven/MavenParsingUtilities.cs

```diff
@@ -57,7 +58,9 @@ private static (string groupId, string artifactId, string version, bool? isDevel
             bool? isDevDependency = null;
             if (results.Length == 5)
             {
-                dependencyScope = MavenScopeToDependencyScopeMapping.TryGetValue(results[4], out dependencyScope) 
+                dependencyScope = MavenScopeToDependencyScopeMapping.TryGetValue(
+                    Regex.Match(results[4], @"^([\w\-]+)").Value,
```

Comment:
Why is this RegEx matching the `-` character along with the word?

Reply:
left over from the pattern I referred. Removed the matching with "-"